README.md

```diff
@@ -0,0 +1,118 @@
+# agentarchives
+Clients to retrieve, add, and modify records from archival management systems
+
+## Installation
+
+### Clone repository
+`git@github.com:artefactual-labs/agentarchives.git`
+
+### Install module
+`cd agentarchives`
+`sudo python setup.py install`
```

Comment:
This can be done in one step using `pip`; maybe that instruction is easier?

```sh
pip install git+https://github.com/artefactual-labs/agentarchives.git
```